BiggerReactors/src/main/java/net/roguelogix/biggerreactors/client/ClientUtils.java

```diff
@@ -0,0 +1,10 @@
+package net.roguelogix.biggerreactors.client;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.resources.IResourceManager;
+
+public class ClientUtils {
```

Comment:
does this need to be in its own file? shouldn't this just be in BiggerReactors.java?